### PR TITLE
Apply style guide to live tab / data views

### PR DESCRIPTION
**Pull request by vwbaker**
Rename functions/variables according to style guide, remove unused headers.

### CODE REVIEW

File: src/OrbitQt/orbitdataviewpanel.h [hunk 1]
```diff
@@ -41,11 +40,11 @@ class OrbitDataViewPanel : public QWidget {
   QLineEdit* GetFilterLineEdit();
 
  private slots:
-  void on_FilterLineEdit_textChanged(const QString& a_Text);
-  void on_refreshButton_clicked();
+  void OnFilterLineEditTextChanged(const QString& a_Text);
```
Comment (antonrohr) [inline src/OrbitQt/orbitdataviewpanel.h:43]:
```suggestion
  void OnFilterLineEditTextChanged(const QString& text);
```

File: src/OrbitQt/orbitdataviewpanel.cpp [hunk 2]
```diff
@@ -9,64 +9,61 @@
...
                                     SelectionType selection_type, FontType font_type,
                                     bool uniform_row_height,
                                     QFlags<Qt::AlignmentFlag> text_alignment) {
-  ui->treeView->Initialize(data_view, selection_type, font_type, uniform_row_height,
-                           text_alignment);
+  ui_->treeView->Initialize(data_view, selection_type, font_type, uniform_row_height,
+                            text_alignment);
 
-  std::string label = ui->treeView->GetLabel();
+  std::string const label = ui_->treeView->GetLabel();
```
Comment (antonrohr) [inline src/OrbitQt/orbitdataviewpanel.cpp:29]: Interesting const placement

I dont know if the style guide actually enforces this, but in [this](https://google.github.io/styleguide/cppguide.html#Constant_Names) section it has is `const type`

```suggestion
  const std::string label = ui_->treeView->GetLabel();
```

**Review by antonrohr — APPROVED** [review]
LGTM, thank you!

File: src/OrbitQt/orbitdataviewpanel.h (hunk 1 above)
Comment (vwbaker) [reply src/OrbitQt/orbitdataviewpanel.h]: done, thanks for catching.

File: src/OrbitQt/orbitdataviewpanel.cpp (hunk 2 above)
Comment (vwbaker) [reply src/OrbitQt/orbitdataviewpanel.cpp]: I like that better as well, visual studio did it wrong :) done.

**Comment by vwbaker** [comment]
(had to rebase for merge conflicts)

**Pull request merged**